perun-scim/src/main/java/cz/metacentrum/perun/scim/api/entities/GroupSCIM.java

```diff
@@ -62,18 +75,59 @@ public void setMembers(List<MemberSCIM> members) {
 		this.members = members;
 	}
 
+	@JsonSetter("id")
```

Comment:
Please remove JsonSetter and getter annotations from both GroupScim and UserScim classes. They are unnecessary because Json serializer can determine these fields automatically. Someone could be later confused why are these annotations used.

Reply:
Ok, I wasn't sure whether it is necessary or not